spring/src/test/java/io/cucumber/spring/contextcaching/SomeTest.java

```diff
@@ -12,12 +12,12 @@
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(classes = {ContextConfig.class})
-public class SomeTest {
+class SomeTest {
```

Comment:
`@RunWith` is a JUnit 4 annotation. So I don't think this will work as expected.